src/main/java/com/iab/openrtb/response/DataObject.java

```diff
@@ -14,5 +14,7 @@ public class DataObject {
 
     String value;
 
+    String label;
```

Comment:
Can you please provide with link to openrtb spec where this field was added? Also, if it was added, you need to update corresponding Protobuf mappers in `ProtobufRequestUtils.java`

Reply:
According to the OpenRTB Native 1.2 the label is absent, but it's present in 1.1. Due to the meaninglessness of the value is used I guess it's done in terms of compatibility in Go. Perhaps we can skip it in PBS Java, what do you think?

Reply:
I meant we could neglect the this part of PBS Go logic

Reply:
Hey, @bretg . Please, take a look at this. Do we need to support `label` field?
[OpenRTB-Native-Ads-Specification-1-1_2016.pdf](https://www.iab.com/wp-content/uploads/2016/03/OpenRTB-Native-Ads-Specification-1-1_2016.pdf)
[OpenRTB-Native-Ads-Specification-Final-1.2.pdf](https://www.iab.com/wp-content/uploads/2018/03/OpenRTB-Native-Ads-Specification-Final-1.2.pdf)

Reply:
I think it's ok to skip an optional Native 1.1 field that was removed in 1.2.